### PR TITLE
Homework setup

### PR DESCRIPTION
**Pull request by nathanallen**
- Scaffolded out the weekly homework.md
- Add `how-tos/homework-submission.md` - draft
- Add `how-tos/request-a-code-review.md`

@tgaff, would appreciate your eyes on this. I think I'd prefer to have submission instructions separate from the homework repo:
 - Easier for us to update/change if needed / ensures students are looking at the same instructions
 - students can focus on setting up their homework forks and worry about the rest later

### CODE REVIEW

File: how-tos/homework-submission.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+- When you begin a lab, you must remember to first `fork` the lab! (Just click the `fork` button).
+- Next, `git clone` your fork of the lab so that you can work on it locally (on your computer).
+- Congrats! You're ready to tackle the lab!
+
+
+### Creating Pull Requests
+When you're ready to submit your homework (and it's okay if you only have a partial solution, we still want to see it!), you need to push your changes to github and create a pull request:
+
+- First, `git commit` your changes
```
Comment (tgaff) [inline how-tos/homework-submission.md]: Might want to mention `git add {filename}`
And git `commit -m `

File: how-tos/homework-submission.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Submitting Lab Work
+Unlike dawn/dusk exercises, *labs* are projects you started in class that you had to fork and clone. These require a slightly different submission flow:
+
+- When you begin a lab, you must remember to first `fork` the lab! (Just click the `fork` button).
+- Next, `git clone` your fork of the lab so that you can work on it locally (on your computer).
+- Congrats! You're ready to tackle the lab!
+
+
+### Creating Pull Requests
```
Comment (tgaff) [inline how-tos/homework-submission.md]: Should this section be moved under Submitting Dawn/Dusk Exercises?  (or made into an h2)

**Comment by tgaff** [comment]
@nathanallen I like the idea of having a "how-tos" directory.
I'll pull the PR instructions from HW.  
I suggested a couple of relatively minor changes, but Good to Merge (GTM) after that.  :cactus:

**Comment by nathanallen** [comment]
Thanks! Improved formatting & clarified git instructions.

**Pull request merged**